Index: openwrt/include/kernel-defaults.mk
===================================================================
--- openwrt.orig/include/kernel-defaults.mk
+++ openwrt/include/kernel-defaults.mk
@@ -34,6 +34,8 @@ ifeq ($(strip $(CONFIG_EXTERNAL_KERNEL_T
   else
     define Kernel/Prepare/Default
 	$(LINUX_CAT) $(DL_DIR)/$(LINUX_SOURCE) | $(TAR) -C $(KERNEL_BUILD_DIR) $(TAR_OPTIONS)
+	$(Kernel/Patch)
+        $(if $(QUILT),touch $(LINUX_DIR)/.quilt_used)
     endef
   endif
 else
